libraries/ashrae/guideline36/4.11-hot-water-plant.ttl

```diff
@@ -0,0 +1,434 @@
+        sh:qualifiedValueShape [ sh:node :primary-loop ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1 ;
+    ] .
+
+:primary-secondary-plant a sh:NodeShape, owl:Class, bmotif:System_Specification ;
+    rdfs:label "Primary Secondary HW Plant" ;
+    sh:class brick:Hot_Water_System ;
+    sh:property [
+        sh:path brick:hasPart ;
+        sh:qualifiedValueShape [ sh:node :boiler ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1 ;
+    ] ;
+    sh:property [
+        sh:path brick:hasPart ;
+        sh:qualifiedValueShape [ sh:node :primary-seconary-loop ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1 ;
+    ] .
+
+# 4.11.1 Boilers
+# TODO add dedicated boiler pump
+
+:boiler a sh:NodeShape, owl:Class, bmotif:HVAC ;
+    sh:or (
+        sh:class brick:Electric_Boiler
+        sh:class brick:Condensing_Natural_Gas_Boiler
+        sh:node :noncondensing-boiler
+    ) ;
+    rdfs:label "Boiler" ;
+    # required
+    sh:property [
+        rdfs:label "Boiler enable" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Enable_Status ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1 ;
+    ] ;
+    # required
+    sh:property [
+        rdfs:label "HW supply temperature setpoint" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Leaving_Hot_Water_Temperature_Setpoint ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1 ;
+    ] ;
+    # optional
+    sh:property [
+        rdfs:label "Boiler firing rate" ;
+        rdfs:comment "Optional point for monitoring boiler firing rate." ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Valve_Position_Sensor ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1 ;
+    ] ;
+    # applications
+    sh:property [
+        sh:path brick:hasPart ;
+        sh:qualifiedValueShape [ sh:node :isolation-valve ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1 ;
+    ] ;
+    # optional
+    sh:property [
+        rdfs:label "Boiler natural gas flow" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Natural_Gas_Flow_Sensor ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1 ;
+    ] .
+
+# TODO topology see comment
+:noncondensing-boiler a sh:NodeShape, owl:Class, bmotif:HVAC ;
+    rdfs:label "Noncondensing Boiler" ;
+    sh:class brick:Noncondensing_Natural_Gas_Boiler ;
+    # applications, required for primary-only plants and primary-secondary plants with noncondensing boilers
+    sh:property [
+        rdfs:label "Plant HW return temperature" ;
+        rdfs:comment "For primary-only plants, locate on plant side of the HW minimum flow bypass to allow for correct load calculations." ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Hot_Water_Return_Temperature_Sensor ] ;
```

Comment:
This is deprecated in Brick now; we are using "Entering"/"Leaving" instead of "Return"/"Supply" for water points. The correct class is `Entering_Hot_Water_Temperature_Sensor`